lib/continuity.js

```diff
@@ -184,7 +184,6 @@ function _getTails({history, witnesses, state = null}) {
   }
 
   const witnessTails = new Map();
-  const witnessSet = new Set(witnesses);
```

Comment:
Here we instantiate a new `Set` for use within the function. The code was changed to use the set passed in as a parameter. Are we modifying the `witnesses` parameter being passed in? Does the function caller expect the same variable to not be modified?

Reply:
No `Set` based mutations seem to be occurring on `witnesses` in this file:

```
andrewjones@Andrews-MacBook-Pro-2 bedrock-ledger-consensus-continuity % grep -inr 'delete(' ./lib/continuity.js  
andrewjones@Andrews-MacBook-Pro-2 bedrock-ledger-consensus-continuity % grep -inr 'clear(' ./lib/continuity.js
andrewjones@Andrews-MacBook-Pro-2 bedrock-ledger-consensus-continuity % grep -inr 'add(' ./lib/continuity.js
./lib/continuity.js:126:    creators.add(_getCreator(e));
./lib/continuity.js:315:      mergeEventHashes.add(event.eventHash);
./lib/continuity.js:338:        parentHashes.add(parentHash);
./lib/continuity.js:982:      fPlusOneSupport.add(tallyResult);
./lib/continuity.js:1043:        union.add(e);
./lib/continuity.js:1056:          union.add(e);
./lib/continuity.js:1061:        union.add(event);
./lib/continuity.js:1108:  supportSets.add(union);
./lib/continuity.js:1247:        endorsers.add(witness);
./lib/continuity.js:1248:        _endorsers.add(witness);
./lib/continuity.js:1369:        priorityPeers.add(witness);
./lib/continuity.js:1381:        priorityPeers.add(witness);
./lib/continuity.js:1388:        priorityPeers.add(witness);
./lib/continuity.js:1397:        priorityPeers.add(witness);
./lib/continuity.js:1406:        priorityPeers.add(witness);
./lib/continuity.js:1419:        proposalEndorsers.add(otherWitness);
./lib/continuity.js:1425:      priorityPeers.add(witness);
./lib/continuity.js:1434:      priorityPeers.add(witness);
./lib/continuity.js:1602:          sortedSet.add(parent);
./lib/continuity.js:1605:      sortedSet.add(event);
./lib/continuity.js:1606:      round.add(event);
./lib/continuity.js:1619:    sortedHashes.add(eventHash);
```

Reply:
If there are no mutations as suggested above, then this change LGTM.